lib/use_case/question/create.rb

```diff
@@ -33,6 +33,9 @@ def call
 
         inbox = ::Inbox.create!(user: target_user, question: question, new: true)
 
+        webpush_app = ::Rpush::App.find_by(name: "webpush")
+        target_user.push_notification(webpush_app, inbox)
```

Comment:
might be better to do this instead so that it doesn't break if the rpush-app is not there yet:

```suggestion
        target_user.push_notification(webpush_app, inbox) if webpush_app
```